distributed/tests/test_worker_metrics.py

```diff
@@ -597,7 +597,7 @@ async def test_new_metrics_during_heartbeat(c, s, a):
         a.digest_metric(("execute", span.id, "x", "test", "test"), 1)
         await asyncio.sleep(0)
     await hb_task
-    assert n > 9
```

Comment:
This `9` feels very magical. since the heartbeat doesn't have to open a new comm, we need fewer ticks until it completes and this is not guaranteed to be past `9`. 

I'm not entirely convinced this test still works/make sense/is relevant.